stories/css-prop-dynamic-object.tsx

```diff
@@ -5,6 +5,10 @@ export default {
   title: 'css prop/dynamic object',
 };
 
+// No idea why I need to add this - but storybook crashes for this stroy if it's not here
+// after adding stories/css-negative-margins.tsx
```

Comment:
Is there an error throw by Storybook? Might be a longshot but maybe because the exported component is named `Styled` in css-negative-margins and it has a title of `styled/negative margins`